scripts/components/pixel-canvas.js

```diff
@@ -35,6 +35,56 @@ class PixelCanvasProto extends HTMLElement {
   }
 
   /**
+    Fetch the current canvas state and turn it into a
+    JSON string representing the canvas.
+
+    First we turn the data into a 2D array where each
+    entry is the color hash of the pixels current
+    background color. Then we JSON.stringify that array.
+
+    @return {string} - a string produced by calling JSON.strigify on
+      a 2D Array containing color hex values, such as '#ffffff' for white.
+  */
+  serializeGrid() {
```

Comment:
Are you talking about line 45 or 48? I believe line 45 is some type of comment format. I thought line 48 was an invoked function - until I saw the opening brace. Good question.